packages/registration_delivery/lib/widgets/progress_bar/beneficiary_progress.dart

```diff
@@ -84,12 +85,13 @@ class BeneficiaryProgressBarState extends State<BeneficiaryProgressBar> {
     final target = targetModel?.targetNo ?? 0.0;
 
     return DigitCard(
-      child: ProgressIndicatorContainer(
+      margin: const EdgeInsets.all(spacer2),
```

Comment:
Why is the margin added?

Reply:
by default no margin is present in new digit component (digit card)